modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-interactive-mode-reference.adoc
+
+:_content-type: CONCEPT
+[id="rosa-sts-overview-of-the-default-cluster-specifications_{context}"]
+= Overview of the default cluster specifications
+
+You can quickly create a {product-title} (ROSA) cluster with the AWS Security Token Service (STS) by using the default installation options. The following summary describes the default cluster specifications.
+
+.Default ROSA with STS cluster specifications
+[cols=".^1,.^3a",options="header"]
+|===
+
+|Component
+|Default specifications
+
+|Accounts and roles
+|* Default IAM role prefix: `ManagedOpenShift`
+
+|Cluster settings
+|* Default cluster version: Latest
+* Default AWS region for installations using {cluster-manager}: us-east-1 (US East, North Virginia)
```

Comment:
@pvasant and @vkareh hi! Can you please help me understand how the default AWS region is determined when installing a ROSA cluster by using the OpenShift Cluster Manager UI?

My understanding is that for cluster installations using the `rosa` CLI, the default region is based on your `aws` CLI configuration. But, for the UI-based installations I'm not sure how this is determined. Is it always set to us-east-1 in the ROSA UI, or is it determined another way?

Thanks!

Reply:
Yes, the UI hard-codes the region to `us-east-1` as the default.

Reply:
Thank you for confirming that.